packages/thirdweb/src/react/web/ui/TransactionButton/index.tsx

```diff
@@ -165,9 +166,9 @@ export function TransactionButton(props: TransactionButtonProps) {
 
           if (onTransactionConfirmed) {
             const receipt = await doWaitForReceipt(result);
-            if (receipt.status === "reverted")
+            if (receipt.status !== "success")
```

Comment:
what other status is there? just curious

Reply:
There's a dropped and replaced status I think but apparently viem doesn't include that so I'm gonna undo this